packages/block-library/src/cover/edit.js

```diff
@@ -301,7 +301,7 @@ function CoverEdit( {
 		setGradient,
 	} = __experimentalUseGradient();
 	const onSelectMedia = attributesFromMedia( setAttributes );
-	const isBlogUrl = isBlobURL( url );
+	const isTemporaryImage = isBlobURL( url );
```

Comment:
I looked at other blocks with images. It seems like they also check for ID. We should probably update the logic here as well for consistency.

Examples:
* Gallery Block - https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/gallery/gallery-image.js#L40
* Image Block - https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/image/edit.js#L61

Reply:
Thanks for the feedback! Yes, adding in the ID check makes good sense to me. I ran out of time to fix this up this week, but I'll take a look at fixing this up next week.

Reply:
Thanks again for taking a look @Mamaduka — I've updated to use an `isTemporaryMedia` function (since this block accepts both images and videos) and renamed the constant to `isUploadingMedia` to be a bit more specific. It's testing well manually for me, but let me know if you'd like any other changes (or would prefer different names for the function / constant).

Reply:
Changes look great. Thanks for the update, @andrewserong.